pkg/minikube/localpath/localpath.go

```diff
@@ -67,6 +67,11 @@ func EventLog(name string) string {
 	return filepath.Join(Profile(name), "events.json")
 }
 
+// AuditLog returns the path to the audit log.
+func AuditLog() string {
+	return filepath.Join(MiniPath(), "logs", "audit.json")
```

Comment:
This should probably be saved in the same directory as `events.json`.

Reply:
I would be okay with it be stored be in "logs" since the nature of audit.json is different (presistant) and event.json is per run and temporary.  but I am okay with different path too ! wdyt @tstromberg ?

Reply:
I agree with Medya, I think it makes sense to leave this in the current directory.